src/common/RefreshNotifier.tsx

```diff
@@ -0,0 +1,25 @@
+import { EventEmitter } from 'events';
+
+export type RefreshParams = {
+  context?: 'HomeScreen';
+};
+
+export const refreshEventType = 'refreshRequested';
+export type RefreshHandler = (refreshParams: RefreshParams) => void;
+export class RefreshNotifier {
+  private emitter = new EventEmitter();
+
+  public addListener(listener: RefreshHandler) {
+    return this.emitter.addListener(refreshEventType, listener);
+  }
+
+  public removeListener(listener: RefreshHandler) {
+    return this.emitter.removeListener(refreshEventType, listener);
+  }
+
+  public emit(params: RefreshParams) {
+    return this.emitter.emit(refreshEventType, params);
```

Comment:
I wonder if there is a way we could make this an async function so that a call could determine when it was all settled. Maybe each listener would return a promise or something. 

Probably don't need it for now but it bugs me that we just have a timeout for the refresh control loading.